python/ray/rllib/models/preprocessors.py

```diff
@@ -24,20 +26,36 @@ def transform(self, observation):
 
 class AtariPixelPreprocessor(Preprocessor):
     def _init(self):
-        self.downscale_factor = self.options.get("downscale_factor", 2)
-        self.dim = int(160 / self.downscale_factor)
+        self.grayscale = self.options.get("grayscale", False)
```

Comment:
run_multinode_tests uses downscale factor currently, so you'll need to fix that for tests to pass

Reply:
Done.